src/vstest.console/Internal/ConsoleLogger.cs

```diff
@@ -155,6 +161,13 @@ protected static IOutput Output
         /// </summary>
         public Verbosity VerbosityLevel => verbosityLevel;
 
+        /// <summary>
+        /// Dictionary of summary of each source. 
+        /// </summary>
+        private ConcurrentDictionary<string, SourceSummary> sourceSummaryDictionary { get; set; }
+
+        private string targetFramework;
```

Comment:
>private [](start = 8, length = 7)

Docs? And also this should move next to the rest of the private members on the top